foundations/installations/prerequisites.md

```diff
@@ -254,6 +254,11 @@ Currently there is a bug preventing CloudReady v83.4 from successfully installin
 
 ### Google Chrome Installation
 
+#### Why Google Chrome?
+
+Because our lessons use Google Chrome, and Chrome/Chromium are overwhelmingly used by devs and consumers, The recommendations we make are very intentional.
```

Comment:
```suggestion
Because our lessons use Google Chrome, and Chrome/Chromium are overwhelmingly used by developers and consumers, the recommendations we make are very intentional.
```